### PR TITLE
[CP-1516][Messages] Importing contacts via MS Outlook is limited to 10 entries and lacks phone numbers

### PR DESCRIPTION
**Pull request by dkarski**
Jira: [CP-1516]

**Description**

[CP-1516]: https://appnroll.atlassian.net/browse/CP-1516?atlOrigin=eyJpIjoiNWRkNTljNzYxNjVmNDY3MDlhMDU5Y2ZhYzA5YTRkZjUiLCJwIjoiZ2l0aHViLWNvbS1KU1cifQ

### CODE REVIEW

File: packages/app/src/contacts/helpers/map-vcfs-to-contacts/map-vcfs-to-contacts.ts [hunk 1]
```diff
@@ -143,7 +184,8 @@ const mapVCFsToContacts = async (files: File[]): Promise<NewContact[]> => {
     }
 
     return contacts.filter((contact) => !isEmptyContact(contact))
-  } catch {
+  } catch (e) {
```
Comment (igorbezsmertnyi) [inline packages/app/src/contacts/helpers/map-vcfs-to-contacts/map-vcfs-to-contacts.ts:187]: ???

Comment (dkarski) [reply packages/app/src/contacts/helpers/map-vcfs-to-contacts/map-vcfs-to-contacts.ts]: I'm joking. Thanks 🙇

**Review by igorbezsmertnyi — APPROVED** [review]

**Review by igorbezsmertnyi — CHANGES_REQUESTED** [review]
Please fix tests

**Comment by dkarski** [comment]
This logic isn't related to the outlook import process. The right fix is here 👉 https://github.com/mudita/mudita-center/pull/923

**Pull request closed without merging**